src/rpc/net.cpp

```diff
@@ -225,12 +225,12 @@ static RPCHelpMan getpeerinfo()
         obj.pushKV("bip152_hb_to", stats.m_bip152_highbandwidth_to);
         obj.pushKV("bip152_hb_from", stats.m_bip152_highbandwidth_from);
         if (IsDeprecatedRPCEnabled("getpeerinfo_addnode")) {
-            // addnode is deprecated in v0.21 for removal in v0.22
+            // addnode is deprecated in v0.21 for removal in v22
```

Comment:
Nit: no need to change unrelated code. I just mentioned the version numbering change as an FYI.

Reply:
Actually, this is incorrect. Current master will become v22. These things can be removed in v23.